cookbooks/aws-parallelcluster-config/libraries/write_chef_error_handler.rb

```diff
@@ -47,14 +47,21 @@ def report
           "mount fsx" => "Failed to mount FSX.",
         }
 
+        # define a mapping from the exception information to the error message we would like to display
+        exception_message_mapping = {
+          "Cluster has been set to PROTECTED mode due to failures detected in static node provisioning" =>
+            "Cluster has been set to PROTECTED mode due to failures detected in static node provisioning."
```

Comment:
Missing "," at the end

Reply:
Should it be a period (".") or comma at the end? (never mind if the original exception message does not include the ending period)

BTW, maybe we can define a variable for the protected mode first to make it slightly more readable? something like this:
```
protected_mode_exception = "Cluster has been set to PROTECTED mode due to failures detected in static node provisioning"
exception_message_mapping = {
  protected_mode_exception => protected_mode_exception + ".",
}
```

what do you think @chenwany ?

Reply:
Oh I was referring to the end of  "node provisioning" in the key part, and maybe you were referring to the missing comma after the whole key-value pair (after "node provisioning.")?

Reply:
Can you also update the screenshot to include the ending period of this error message sentence @ohchocolate ?

Reply:
Yeah my comment is about adding `,` after `."`, about the formatting issue
Yes I think Delong's comment make sense too